simulationworkflowschema/molecular_dynamics.py

```diff
@@ -384,6 +392,145 @@ class BarostatParameters(ArchiveSection):
     )
 
 
+class Lambdas(MSection):
```

Comment:
maybe we always use ArchiveSection, I think it does not hurt

Reply:
done.